crates/libs/core/rustfmt.toml

```diff
@@ -0,0 +1 @@
+newline_style = "Unix"
```

Comment:
I believe our `.gitattributes` configuration makes this particular attribute moot.

If you're asking about the funky rustfmt settings in general, we use a high column count to significantly reduce the size of the crate(s). I suspect this is why the PR only targets a few crates.

Reply:
Ah. That was indeed confusing me. That seems like a good reason to me but it'd be nice to call that out somewhere.

Reply:
Also, what about doing the reverse? Have the increased width set on crates where it matters but not globally?

Reply:
Let's do that then. Add column width tomls into `windows` and `sys` (am I missing any?) and let the other crates pick up the root configuration.

Reply:
Agreed.  I've added rustfmt.toml files to windows and sys dirs, and removed them everywhere else.  I'm certainly happy to go with the more aggressive (and future-looking, in case new crates are added) change.